.gitignore

```diff
@@ -1 +1,2 @@
-.DS_Store
\ No newline at end of file
+.DS_Store
+.idea/
```

Comment:
Do we really want to do this? Pet shop is in the instruction repo and has a `.idea` folder that needs to be maintained.

Reply:
This has been addressed in commit 39ed688d24a0452360350df7f757ffe09501bd8f.

See the screenshot of the `git` system correctly recognizing new files in the `petshot/.idea/` folder (it already accepted changed files without this change).

<img width="643" alt="image" src="https://github.com/user-attachments/assets/8e6df1ad-2de6-4fa2-8d16-a5f8ccede89c">